tests/data/test303

```diff
@@ -37,7 +37,7 @@ https
 HTTPS with 8 secs timeout
 </name>
 <command>
--k https://%HOSTIP:%HTTPSPORT/want/%TESTNUMBER -m 8
+-k https://%HOSTIP:%HTTPSPORT/want/%TESTNUMBER -m 3
```

Comment:
This was last increased (to 8 sec) in commit e2fc294 with the comment *added an additional second to allow test to pass on heavily loaded servers*. Are you sure it's safe to drop it this low? Whatever it is, the name on line 37 needs to be changed to match.

Reply:
Thanks for the info. I'll revert this.